pizza_cutter/des_pizza_cutter/integration_test/data.py

```diff
@@ -278,10 +284,48 @@ def generate_sim():
         'dvdy': wcs_scale,
         'x0': 24,
         'y0': 24}
+
+    # the first number must match the "central_size" and the second must match
+    # "buffer_size" in the config in in test_coadding_end2end.py those are
+    # currently hard coded into the test, so you cannot change these
```

Comment:
Thank you for these comments. I should have made them myself and I am sorry you had to figure it out. That mustr have been frustrating and I really appreciate your patience.

Reply:
you're welcome, no problem